src/moscot/base/problems/compound_problem.py

```diff
@@ -237,87 +278,87 @@ def prepare(
         )
         self._problem_manager.add_problems(problems)
 
+        # we assume that all subproblems are of the same kind
         for p in self.problems.values():
             self._problem_kind = p._problem_kind
+            self._stage = "prepared"
             break
         return self
 
     def solve(
         self,
         stage: Union[ProblemStage_t, Tuple[ProblemStage_t, ...]] = ("prepared", "solved"),
         **kwargs: Any,
-    ) -> "BaseCompoundProblem[K,B]":
-        """
-        Solve the biological problem.
+    ) -> "BaseCompoundProblem[K, B]":
+        """Solve the individual :term:`OT` subproblems.
+
+        .. seealso:
+            - TODO(michalk8): link an example/tutorial
```

Comment:
@ArinaDanilina please open an issue about this

Reply:
#565